tools/mkmapdata/mkmapdata.sh

```diff
@@ -339,11 +339,15 @@ case $hostname in
   if [ -z "$REGRID_PROC" ]; then
      REGRID_PROC=36
   fi
```

Comment:
The following changes were originally discussed in this post:
https://github.com/ESCOMP/CTSM/pull/823#issuecomment-578277067